### PR TITLE
compiling fix

### PR DESCRIPTION
**Pull request by casotoroar**
i think i did it

### CODE REVIEW

File: source/PlayState.hx [hunk 1]
```diff
@@ -46,22 +47,37 @@ import shaderslmfao.BuildingShaders.BuildingShader;
 import shaderslmfao.BuildingShaders;
 import shaderslmfao.ColorSwap;
 import ui.PreferencesMenu;
+import StrumNote;
+
+#if hxCodec
```
Comment (casotoroar) [inline source/PlayState.hx:52]: use #if hxcodec instead

(dont use caps for hxcodec)

**Comment by MemeHovy** [comment]
no

**Comment by MemeHovy** [comment]
you didn't

File: source/PlayState.hx [hunk 2]
```diff
@@ -930,22 +993,55 @@ class PlayState extends MusicBeatState
 		super.create();
 	}
 
-	function ughIntro()
+	function playCutscene(name:String, atEndOfSong:Bool = false)
 	{
+		#if hxCodec
```
Comment (casotoroar) [inline source/PlayState.hx:998]: same here: #if hxcodec

**Review by casotoroar — COMMENTED** [review]
yeah uhh check these over. i think i did it right

File: source/PlayState.hx (hunk 2 above)
Comment (casotoroar) [reply source/PlayState.hx:998]: also make two new folders, scripts and shaders inside of assets.

add a readme or something inside of it doesnt really matter

**Comment by casotoroar** [comment]
not too much of a coder but i tried ig

**Comment by MemeHovy** [comment]
but you added things that would be in fork

**Comment by MemeHovy** [comment]
and it's not even your own fork 🤦🏻‍♂️
you're such a fucking retard

**Comment by ACrazyTown** [comment]
> and it's not even your own fork 🤦🏻‍♂️
> you're such a fucking retard

Calm down bro, yeah its an unneccesary PR that wont get merged but no need to be so pressed got damn!!!

**Comment by MemeHovy** [comment]
> > and it's not even your own fork 🤦🏻‍♂️
> > you're such a fucking retard
> 
> Calm down bro, yeah its an unneccesary PR that wont get merged but no need to be so pressed got damn!!!

sorry I was very irritated at the time I wrote that

**Comment by RandomUserOnInternet** [comment]
Bro There some code from Psych Engine 💀

**Comment by Ya-boy-Jotaro-KuJo** [comment]
please close this frankenstien thing

**Comment by Wither362** [comment]
Hahahaha!

This is a **real** pull request! Yes dude!

**Pull request closed without merging**